Codegen/test/Codegen.Test.csproj

```diff
@@ -93,6 +93,9 @@
       <Link>resources\config\TypeCobol\skeletons.xml</Link>
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Include="resources\output\TypeCobol\Global_Storage\GlobalStorage.rdz.tcbl.lm">
```

Comment:
Can you remove this tag?
Because this file is duplicated in the project due to the previous rule:
```
<None Include="resources\output\**">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>	      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>	    </None>
```